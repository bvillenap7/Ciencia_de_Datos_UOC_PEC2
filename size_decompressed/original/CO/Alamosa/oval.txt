I am a police officer in alamosa colorado. on the date and time above, i was on routine patrol, i was traveling northbound along a coun;1998-12-09 01:30:00;45 seconds;I am a police officer in alamosa colorado. on the date and time above, i was on routine patrol, i was traveling northbound along a county road I saw a light in the sky and i looked up and saw a very large green light falling toward the earth. the light was very bright green. at first i thought it was a falling star. i continued looking as it fell below the horizon. my partner saw what he described as the same thing that i saw two night later in a different area in the sky;37.4725;-105.8796
