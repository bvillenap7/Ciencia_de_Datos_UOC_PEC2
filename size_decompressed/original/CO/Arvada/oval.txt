Daylight disc , Jeffco Colorado , 2001;2001-11-10 15:00:00;30 seconds or so;Daylight disc , Jeffco Colorado , 2001 In the fall of 2001 , my elderly father and I were attending a high-school football playoff game on a bright late-fall Sunday afternoon at the beautiful North Jeffco stadium , nestled right against the foothills of the Colorado front range . At one point , a jet roared overhead , and many spectators immediately looked up ( a not-unusual reaction immediately following the 9/11 debacle ! ) and saw a 757 heading west at about 5000 ft. altitude . Everyone looked back down to the action on the field...and my father goes  What the hell is THAT ?  . I looked back up , and on a south to north heading , at a hieght of no more than a thousand or so feet , a small , oval or egg shaped , brightly reflecting silver object was zipping at a good clip , disappearing to the north in a matter of less than half a minute . My Dad was in WWII , well versed in estimating distance , speed , and aircraft identification , and I have been an aircraft watcher and fan for over 35 years.......and niether of us have ever seen anything to compare with the high speed or odd shape of this problematical object . ((NUFORC Note:  Date may be approximate.  PD));39.8289982352941;-105.110888235294
During 4th of July event, we saw four orange lights moving very slow and smoothly in the sky appearing and disappearing one at a time.;2014-07-04 22:00:00;3 minutes;During 4th of July event, we saw four orange lights moving very slow and smoothly in the sky appearing and disappearing one at a time. On July 4th, 2014, at about 10 pm mountain time.  I was watching the firework display in Arvada Colorado.  In the east, off to the right of the fireworks, I saw four orange yellowish lights slowly moving to the right, and then moving to the left in perfect sink.  At first I thought it was a part of the show, but after about a minute of watching, I told my wife and she agreed that it couldn't be a part of the show.  We both watched as the lights moved very slowly from side to side.  Then, while in continued slow smooth motion, one light disappeared and then another light disappeared, and then another, until one light remained. It remained in continued slow steady motion. The other lights then reappeared one at a time until all four lights appeared. Once again, they all appeared to be moving slow and steady from side to side in perfect sink. After about another minute the lights began to disappear one at a time until one light remained. The remaining light continu! ed movement but seemed to dip downward a bit before completely disappearing from site.;39.8289982352941;-105.110888235294
Mysterious lights moving back/forth and up/down in the treeline.;2014-08-18 22:00:00;30 minutes;Mysterious lights moving back/forth and up/down in the treeline. Monday, August 18 in Arvada, CO, (near Standley Lake) – mysterious lights moving back/forth and up/down between 10:00 pm and 10:30 pm.  Sitting on my back patio I noticed a blue/red/white lights moving erratically due north, about 20 degrees above the horizon (near the base of the Big Dipper). The lights moved in a fashion similar to somebody pointing a laser pointing at a wall and moving it around erratically.  However, these lights were much bigger and were blue/white/red, depending on which way the lights were moving. From my back porch, I sit about even with the treeline.  I could see a blinking light moving up/down and back/forth up out of the treeline, and above/down into the treeline.  The lights would do this for about 30 seconds, then stop for about 5 minutes , then start again.  The happened for about 30 minutes. At about 10:15, I also noticed, what looked like, a shooting star heading in the general direction of the moving lights.  I thought it was a shooting star, until I realized there was some cloud cover in that direction, so the “shooting star” had to be in front of the cloud cover (which I thought was strange)  At about 10:30 pm, I got in my car and drove about a mile north to get a better view, overlooking Standley Lake.  However, by this time the lights had disappeared.    The sighting was in the general direction of Rocky Mountain Airport.;39.8289982352941;-105.110888235294
Four UFOs flying in diamond pattern seen over Arvada, CO.;2015-11-01 18:08:00;5 minutes;Four UFOs flying in diamond pattern seen over Arvada, CO Hi there! My boyfriend (a police officer) and I (molecular lab tech) had both just left my house in Arvada, driving in two different directions and we both saw four, slow-moving, almost hovering oval shapes in the sky that definitely were not helicopters or planes.   They were moving very close together in a diamond shape and were emitting a yellowish-white light.  The light of one of them would fade slowly in and out, as though it was possibly disappearing behind a cloud.  I tried to follow them toward the east but lost sight and they disappeared.   This is my first ever sighting.   I was able to take a picture on my mobile phone of two of them due to obstructed view.;39.8289982352941;-105.110888235294
Zoomes left to right then still as the red flashing light just sits in mid air, then it zoomes away very quickly.  ((anonymous report));2016-05-01 21:00:00;Just sitting on patio;Zoomes left to right then still as the red flashing light just sits in mid air, then it zoomes away very quickly behind the trees Me and 2 other people were just relaxing at around 9 p.m on our patio and it was dark so we were just sitting looking at the sky then i see a red blonking light and just stood still then zoomed left and and right then went still again and then just zoomed away behind the trees.  It wasn't a plane, planes dont stop in mid air and start going left to right and also planes make noises this thing made no niose.  Lastley it seems many people have been sighting almost the same thing i have. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.8289982352941;-105.110888235294
