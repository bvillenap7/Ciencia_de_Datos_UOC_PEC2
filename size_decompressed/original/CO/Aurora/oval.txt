Brillant oblong object spotted over Aurora, CO.;2009-09-16 13:20:00;15 seconds;Brillant oblong object spotted over Aurora, CO. It was a clear, Sunday afternoon.  No clouds.  I was sitting in the kitchen have lunch. My sliding glass window faces my backyard.  My backyard backs up to a park. I live in a new development. The trees are young, not very big. At approximately 13:20 hours (or 1:20 ish) I saw a oblong/oval brillant object in the sky. It appeard like it was floating. There was no noise. It was the brightest and most unusual object I have ever seen. I live very close to Buckley Airforce base. I am very familar with the jets/planes that fly around as well as the very loud noise they make.  The object was not very big, although it was hard to gage the distance.  It probably was the size of an F14 - just oval and made of light. It was sun brillant.  I could make out by the shadows that it appared to have two decks. I watched as it moved behind a small tree.  I waited to it see appear on the other side, never looking away, but it never reappeared on the other side of the tree. I w! ent outside to look in all directions and it just vanished.  About 7 to 10 minutes later, I saw a jet (white and red not grey like all the other jets I've seen) take off in a southeast direction. Just one jet, no others followed.  I know what I saw was real.;39.6695444272445;-104.803991486068
UFO spotted in SE skies this past summer.;2012-07-14 21:00:00;<5 minutes;UFO spotted in SE skies this past summer. We were watching a movie in the backyard at night when one of our guests told us all to look at the sky.  It had a light blue light and was about the size of a pea from where we were sitting.  It was stationary for a second or two and then moved in zigzag directions before it descended out of our site. We thought for sure it was a UFO and called Buckley Air Force Base.  The gentleman that answered the phone stated he did not hear of anything and responded to us as if we were crazy for asking. We knew it was not a plane, jet, or helicopter and before this day I was skeptic of so called UFO sightings. I am not for sure of the date this happened but it was in the summer of this year and was approximately 3 nights prior to the news story of the pilot who called his air traffic control people to state he just witnessed a UFO in flight. I had never heard of this site before until I just read the news story from November 8th, of the UFO sighting in Denver.  Now it really has me curious as to what we saw.  The object was in the South-West sky.  The observers of this UFO were 4 families, including 4 children (ages 5 years old) who were also excited about seeing the object, asking what it was they had just witnessed. ((NUFORC Note:  Witness indicates that the date is approximate.  PD));39.6695444272445;-104.803991486068
