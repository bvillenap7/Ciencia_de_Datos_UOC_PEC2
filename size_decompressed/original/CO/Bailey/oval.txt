I was listening to your show late last night( about 2:30 AM ) while doing my computer work. Before going to bed I went out to check the;2002-06-10 02:30:00;3 min;I was listening to your show late last night( about 2:30 AM ) while doing my computer work. Before going to bed I went out to check the horses and feed a little hay. As always I look up at the stars but the clouds had rolled in making star watching nearly impossible. A few starts shown through the clouds in between the breaks. As I was walking from the barn with hay to throw to the horses when I notice to the North this bright blue light coming down from the sky in an odd snake-like fashion, disappearing behind the hills miles away. Less then 5 minutes later it reappeared and was climbing back into the sky in the same snake-like motion to the North and West. There was enough of a break in the clouds to see this. This thing was huge, bright and bluish in color and had no tail like a meteor would have. It skirted about the hill tops just west of here... I couldn't make out any shape other then a bright blue light. This afternoon I was talking to a friend of mine who is also a night owl computer person and he was out doing likewise at 3AM over in Rivertown Utah. He described the same thing but it was to the east of him and over the Wassatch Mts. he had nearly clear skies from his point of view. I have a friend who works  as a Country Western personality for Waitt Media in Vail Colorado. I take care of his place here in Bailey Colorado and communicated what I saw last night in my daily eMail to him. He said the News Services carried a segment on an object seen over Utah, Wyoming and Colorado.;39.4497;-105.4692
