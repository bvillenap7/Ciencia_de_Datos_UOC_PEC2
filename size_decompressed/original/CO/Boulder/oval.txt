My Girl-Friend and I noticed something strange in the sky above Boulder, Oval in shape and I have a Video Tape of it.. Interested??;1999-07-15 15:00:00;10-15 mins.;((NUFORC NOTE:  A RUMOR IS CIRCULATING THAT A PERSON CAN SEE AND PHOTOGRAPH UFO'S BY LOOKING AT, AND POINT A CAMERA IN THE DIRECTION OF, THE SUN.   WE HAVE SEEN NO EVIDENCE THAT THIS ASSERTION IS TRUE!!   MOREOVER, WE STRONGLY ENCOURAGE READERS HERE TO NEVER LOOK DIRECTLY AT THE SUN, AND, IN PARTICULAR, TO NEVER LOOK AT THE SUN THROUGH AN OPTICAL DEVICE.  Peter)) My Girl-Friend and I noticed something strange in the sky above Boulder, Oval in shape and I have a Video Tape of it.. Interested?? Last Year I started a Video of my Daughter her Friend and my Girlfriend. I enjoy reading www.sightings.com and decided to try Jon-Bro's Techinques, Which are also on the film.. This year, approx: July 15th I walked outside on the front steps to have a cigarette and noticed a strange looking object in the sky. It was positioned approx: South-southwest at about position 09:00 o'clock in the sky, just in the back-drop of the Flatirons mtns. I yelled to my Girl-friend to look at what I was seeing. She came out-side and I told her to look.. At this time the object was Due South and about at position of: 11:00 o'clock. It hovered there for approx: 10-15 mins., as if it was looking at us. During this time I ran in to get the Video-Recorder.. The Batteries where nearly dead which accounts for the lack-of-ability to Zoom.. However I managed to get approx: 10-15 mins. of Video on the object that just hovered before it dissapeared..  The Object at first appeared Square, then appeared Oval with dim lights on the sides of the Oval. I tried to get a picture of the house below the object to get an approx: size of the object.. I fumbled several times with the Video camera due to checking the power which I plugged in later. I and my Girl-Friend were witnesses..  I have the Video of this if anyone is intersted..;40.0217727272726;-105.258365102639
The metal oval objects were there and then they were gone without a trace!;2002-03-31 17:30:00;30 Seconds;The metal oval objects were there and then they were gone without a trace! A friend and I were driving north on hwy 93 to boulder from I-70 when I noticed an object in the sky with clouds in its background. The object had an odd reflection as if it were made of chrome, and it was not moving in any direction at all. So I pointed it out to my friend and he spotted another one just a little higher in the sky with just a blue sky background. So I say to my friend, I think those are spaceships? Then without a blink they disappeared with no trace. It was wierd because they just were there and then were gone. They both looked like horizontal ovals of chrome or some type of reflective metal, and had nothing to campare to a regular plane or jet. That was the wierdest and most unreal 30 seconds of my life.;40.0217727272726;-105.258365102639
October 1, 2006 meteor reported across skies of Arizona, New Mexico, and Colorado;2006-10-01 23:15:00;30 sec;October 1, 2006 meteor reported across skies of Arizona, New Mexico, and Colorado I was watching TV. The lights were out the window shades were open. I noticed a  white movement out the window. I jumped out of bed saying What it that? What isss that? My husband stirred. It looked like 4-5 very large thin oval shapes, directly in a line, low in the sky heading east, I was looking south. The movement was just slightly south/east. There were long thing clouds in the sky, it was clear. It seemed bright- mostly from the not full moon but also from the light. There was no sound. I said, look, oh wait you missed it (the items had just disappeared behind the siloette of a tree. Wait, look to the left of the tree you will see it come out...nothing did.  I couldn't believe it. I descibed this to my husband. The next few days I could not let it go. I went to a UFO sightings website (something I had never done before)and looked for sightings. Several people descibed seeing something similar at about the same time and day. I did not report my sighting. But then, Saturday, I read the Friday Colorady Daily newspaper, where there was an article about a man who is a meteor hunter. It stated he searched for marble-size rocks in a 50-mile strip of Colorado.There was a picture of him holding a piece of the meteorite that occured on Oct. 1, 2006. One of the brightest meteors reported in recent years slow-danced across Arizona, New Mexico, and Colorado. Apparantly landing south of Colorado Springs. And he was holding a piece of it that looked about the size of a coconut. ((NUFORC Note:  Witness elects to remain totally anonymous  provides no contact information.  PD));40.0217727272726;-105.258365102639
((HOAX))  TRUE HIDDEN EVENTS;2009-12-01 20:00:00;ALL-NITE;TRUE HIDDEN EVENTS READ PLANET X THEORY/FACT OR FICTION(NOBLES/GABLESBOOK STORE)SOMETHING THAT NO ONE HAS BOTHERED TO TALK ABOUT ANYMORE, TO OPEN THE PEOPLES EYES ABOUT COMING EVENTS. NOT TO PLAY WITH ANY ONES MINDS,WHY DID MOVIE MAKERS MAKE THE MOVIE 1212 BUT MADE MOVIE ABOUT EVENTS COMING FROM WITH-IN THE EARTH, BULL@#$- PLEASE READ THIS BOOK THEN YOU MAKE YOUR MIND UPYES THE GOVERMENT IS HIDING THE TRUTH ABOUT UP COMING EVENTS LOOK UP TO THE SKY FOR THE TRUTH AN BELIEAVE ASK YOUR SELF A QUESTION WHY IS IT THAT THE AZTEC INCAS HAD A DRAWINGS OF THE PLANETARY SYSTEMS ON THEIR DRWINGS WAY BEFORE TELESCOPES WERE INVENTED??? (READ THE BOOK FOR INFORMATION)IN THIS BOOK YOU WILL KNOW WHY THIS BOOK IS CALLED PLANET-X-  REMEMBER I BELIEAVE THAT THE GOVENMENT IS NOT TELLING THE TRUTH.;40.0217727272726;-105.258365102639
