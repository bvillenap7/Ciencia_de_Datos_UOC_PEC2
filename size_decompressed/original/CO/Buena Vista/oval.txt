White oval with gold light/energy over mountain in daylight;1997-03-15 10:30:00;10 mins;White oval with gold light/energy over mountain in daylight It was a clear day and my girlfriend and I were letting our dogs run around the rodeo grounds near Buena Vista Co. This is in a valley with 12 to 14 thousand foot mountains running North-South. The Collegic Peaks were south of us, also near Mt. Antero.  Looking northeast from rodeo ground right above mtn. peak was a white oval shaped object hovering stationary.  It was approximately a couple miles away and perhaps 2-3 times bigger than a goodyear blimp. It was white, oval shaped.  I had binoculars in the car and so used them for a better look. I could get the peak of the mountain in focus but the object remained a bit fuzzy with a gold light shimmering inside of it. This was as if it was emitting some sort of energy or field around it that sort of distorted (like heat waves).  However it was still clear enough to watch for about ten minutes w/ binoc's and w/out.  Closest thing it could have been was a blimp, but of course wasn't in this remote location. It then just vanished no trail, trace, anything just poof- gone like into another dimesion or something. remember clear daylight 10-11am or so. not typical saucer or round sphere-OVAL and just vanished. ((NUFORC Note:  Witness indicates that date of sighting is approximate.  PD));38.830028;-106.14436
