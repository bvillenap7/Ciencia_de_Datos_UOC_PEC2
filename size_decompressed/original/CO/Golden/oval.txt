Spot moving on the moons surface;1997-07-15 00:00:00;2 min;Spot moving on the moons surface After many nights of using a fairly high power lense on my reflection telescope I focused on the moons surface.  After learning to track the moon as it moved rapidly across my field I started focusing on the surface.  I used the craters as the reference as where to focus.  After looking at the area on the periphery of the lighted portion of the moon (I used this because it seemed easier to actually see terrain distinctly and in focus). Almost immediately I noticed an object traveling North by North East (upper right direction as my vision would orientate.  It traveled in a precisely straight line for close to one minute.  During this time I tried to determine if this was a just a surface phenomenon or something on my lens.  I quickly determined that it had to be on the surface as it did not appear until I had complete focus on the surface craters and features. After I had established that fact, and continued following the object it stopped for a few seconds (I mention this only because it was a distinct period however short).  It then turned left (almost) 90 degrees to the left and traveled for another 5-10 seconds before stopping.  It remained at that position for well over a minute or so.  I followed it until the moon disappeared behind the trees and the house I lived in - Mountains of Colorado. I tried again in the back of the house (to bypass the blockage that the house and trees had presented) but was unable to establish exactly what or where the spot was. My observation was that the shadow did not elongate as a shadow would if it were to rise and fall on the landscape (because it was on the rising or setting light source).  It remained in clear focus during the entire time and also remained as a spot and did not become an elongated feature or blurred image.;39.7598265306123;-105.220002040816
